src/libpriv/rpmostree-core.c

```diff
@@ -3813,15 +3813,15 @@ rpmostree_context_assemble (RpmOstreeContext      *self,
   }
 
   guint overrides_total = overrides_remove->len + overrides_replace->len;
+  const char *progress_msg = "Package checkout";
```

Comment:
Minor: how about just "Installing packages" since we consistently use the present participle elsewhere? I think the "Applying overlays" message might date from before we ran scriptlets.

Reply:
I thought about that, but we're different from yum/apt where we run scripts in a separate phase; felt like it was worth using slightly different terminology to call that out?   I don't mind changing though if you prefer.

Reply:
How about "Checking out packages"?